docs/grammar.md

```diff
@@ -127,7 +127,7 @@ struct = unit_struct | tuple_struct | named_struct;
 unit_struct = ident | "()";
 tuple_struct = [ident], ws, tuple;
 named_struct = [ident], ws, "(", [named_field, { comma, named_field }, [comma]], ")";
```

Comment:
```suggestion
named_struct = [ident], ws, "(", ws, [named_field, ws, { comma, ws, named_field, ws }, [comma, ws]], ")";
```

Reply:
the extra `ws` by the commas are unnecessary, `comma = ws, ",", ws`

Reply:
Ah sorry, you’re absolutely right. I think my edit should have corrected it now?

Reply:
Looks good. Maybe comb the rest of the definitions as well.